.github/label-actions.yml

```diff
@@ -1,7 +1,14 @@
 # When `devnet-test-e2e` is added, also assign `devnet` and `push-image` to the PR.
 devnet-test-e2e:
   prs:
-    comment: The CI will now also run the e2e tests on devnet, which increases the time it takes to complete all CI checks. If you just created a pull request, you might need to push another commit to produce a container image DevNet can utilize to spin up infrastructure. You can use `make trigger_ci` to push an empty commit.
+    comment: |
+      The CI will now also run the e2e tests on devnet, which increases the time it takes to complete all CI checks.
+
+      You may need to run `make trigger_ci` to submit an empty commit that'll trigger the tests.
+
+      [GCP workloads (change the namespace)](https://console.cloud.google.com/kubernetes/workload/overview?project=protocol-us-central1-d505)
```

Comment:
Can you do one of the following:

1. Tell the developer what to change the namespace to by using `{issue-id}` below
2. Just make the link automatically point to the correct devnet namespace.

Option 2 is preferrable if possible.

Reply:
@okdas ☝️

Reply:
@Olshansk I did both, but this link, unfortunately, will only work for somebody who either only has one Google account or the first account they logged into is `grove`. The reason is the `authuser=ID` argument that is required when you're logged into many accounts. I personally won't be able to use this link. I assume this is the case for other people as well, hence I mentioned the need to change the namespace.